runtime/jvmti/jvmtiHook.c

```diff
@@ -2139,8 +2151,10 @@ jvmtiHookPopFramesInterrupt(J9HookInterface** hook, UDATA eventNum, void* eventD
 static void
 jvmtiHookResourceExhausted(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
 {
-	J9JVMTIEnv * j9env = userData;
+	J9JVMTIEnv *j9env = userData;
+	J9VMVmResourceExhaustedEvent *data = (J9VMVmResourceExhaustedEvent *)eventData;
 	jvmtiEventResourceExhausted callback = j9env->callbacks.ResourceExhausted;
+	J9VMThread *currentThread = data->currentThread;
```

Comment:
since `data` is used only once
```suggestion
	J9VMThread *currentThread = ((J9VMVmResourceExhaustedEvent *)eventData)->currentThread;
```

Reply:
Please apply Babneet's suggested change.

Reply:
`data` is not used once.